backend/routes/articles.js

```diff
@@ -26,6 +26,30 @@ router.post('/', (request, response) => {
 
   newArticle
     .save()
+    .then((newArticle) => response.json(newArticle))
+    .catch((err) => response.json('Error:' + err));
```

Comment:
Make sure to send an HTTP status code to tell the client, that the request failed.
Same issue in other requests.

```suggestion
    .catch((err) => response.status(500).json('Error:' + err));
```